.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
+install:
+  - pip install pandas
+  - pip install codecov
+# command to run tests
+script:
+  - coverage run -m unittest discover
+after_success:
+  - bash <(curl -s https://codecov.io/bash) -t b5b039a8-7446-4655-b25e-bda6f3168fd7
```

Comment:
The private codecov token for flux-accounting should not be checked in to your repo in the clear.

Instead set `CODECOV_TOKEN` as a "secret" environment variable in the Travis-CI settings page for flux-accounting:

https://travis-ci.com/github/flux-framework/flux-accounting/settings

Then that variable and your private codecov token will not be exposed to anyone except maintainers.

Reply:
Thanks Mark. I believe I was able to set that environment variable correctly and pushed those changes up. Let me know if this is what you were thinking of. 

Looks like Codecov said that no coverage was uploaded, but looking at the [Error Reference](https://docs.codecov.io/docs/error-reference#section-missing-base-commit), I am wondering if it is because these changes haven't landed on the `master` branch yet?

Reply:
Yeah, since there is no coverage report for the main branch, codecov cannot report changes. That is expected.